src/components/Contributors.js

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from "react"
 import styled from "styled-components"
+import _ from "lodash"
```

Comment:
```suggestion
import { shuffle } from "lodash"
```
Let's import the function directly, since it's the only one we're using.